### DISCUSSION TITLE
nice framework to make api with db and orm, but any admin ui lib?

### DISCUSSION THREAD

**Discussion by yeukfei02** [adonisjs/core#2448] [Help]
btw let's say if I want to make an admin portal for this API/backend project.
is there are nice admin lib can use?
I google search looks like there are not many options.

thanks.

**Comment by yeukfei02** [comment]
if using ruby on rails, there are `activeadmin` and `administrate` can use

**Comment by RomainLanz** [comment]
Hey @yeukfei02! 👋 

I don't know any community Admin Dashboard built on top of AdonisJS and there is no plan to make an "official" one.
You will need to create your own.
